package.json

```diff
@@ -44,7 +39,12 @@
   },
   "devDependencies": {
     "@stoplight/scripts": "^5",
+    "@stoplight/types": "^11.4.0",
```

Comment:
This might break usage of this repo for anyone that doesn't have @stoplight/types installed since we're using const enums. Can you verify it works without installing @stoplight/types?

Reply:
I'll move this back in the dependencies then, checking this out in all the repo is a lot of work and not worth